sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -14,6 +14,39 @@ load(
 
 package(default_visibility = ["//visibility:public"])
 
+opentitan_functest(
+    name = "rom_e2e_shutdown_exception_c",
+    srcs = ["rom_e2e_shutdown_exception_c_test.c"],
+    cw310 = cw310_params(
+        bitstream = "//hw/bitstream:rom",
+        # Note: `exit_failure` should actually be something like
+        # "(?!ROM:[0-9a-f]|BFV:01495202)" but the crate that we use does not
```

Comment:
(Warning, I'm about to be pedantic and I'm probably subtly wrong.) I'm a little confused by this proposed regex. I think the intent is to fail fast when you see something that isn't `ROM:[0-9a-f]` and isn't `BFV:01495202`, but I'm not sure the negative lookahead would do that.

I understand `a(?!b)` matches `a` so long as it's not followed by `b`. I believe `(?!b)` would match empty strings that aren't followed by `b`. If the input were `b`, then `(?!b)` could still match the empty string after `b`.

Specifically, if the input were `ROM:1`, wouldn't the regex happily match the empty string between `O` and `M`, causing the test to fail even on the successful output?

Reply:
You're right, this doesn't really make sense. I was trying to put something together but couldn't verify since the crate doesn't support this :) Removed.

Reply:
I suppose you could approximate the fail-fast behavior by setting `exit_failure` to the set of characters that would never be matched by `exit_success`. Well, assuming that `exit_success` is meant to completely match the output, which probably isn't true.